CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.3.0] 2020-09-01 (UNPUBLISHED)
+
+- We converted the ChannelList component from a class to a function. We also abstracted the event listener logic into custom hooks. On upgrading to this release, ensure events and any custom handling functions (ex: `onAddedToChannel` or `onMessageNew`) are properly processed and update the UI as expected.
+
```

Comment:
Lets add a detailed explanation - similar to https://github.com/GetStream/stream-chat-react/blob/master/CHANGELOG.md#110-2020-06-08

Also, please add the breaking changes regarding context:

1. Which new contexts are introduced
2. Which context values are moved from where to where